ProteoformSuiteGUI/ExperimentExperimentComparison.cs

```diff
@@ -74,6 +83,29 @@ public DataGridView GetEEPeaksDGV()
             return dgv_EE_Peaks;
         }
 
+        public bool get_notch_masses()
+        {
+            try
+            {
+                string[] notch_masses = tb_notch_masses.Text.Split(';');
```

Comment:
If you check notch search, the tb says "enter notches to search separated by semicolon"